### ISSUE TITLE
loss of training does not descrease

### ISSUE THREAD

**Issue by babak-abad** [tuanphan09/captcha-recognition#6]
I'm training your network. I test different values for lr(0.01, 0.001, 0.0001, 0.00001) but loss value does not decrease from 0.07 and validation loss does not decrease from 0.002. Accuracy is 88%. I also test with batch_size = 200, 250, 300 but It is still unchangeable. How can I fix this problem? Any help or suggestion will be appreciated.

**Comment by tuanphan09** [comment]
It's hard to get 100% accuracy, the loss would converge and stuck at some value - 'local optima'. In general, there are many things you can try: changing model complexity, data pre-processing, data augmentation, etc.
